docs/plugins/github-apps.md

```diff
@@ -89,6 +89,26 @@ integrations:
         - $include: example-backstage-app-credentials.yaml
 ```
 
+### Limiting the GitHub App installations
+
+If you want to limit the GitHub app installations visible to backstage you may
+optionally include the `allowedInstallationOwners` option.
+
+```yaml
+appId: 1
+allowedInstallationOwners: [1234]
```

Comment:
Update this to be some dummy name instead? 😅

Reply:
Done

Reply:
Done